.eslintrc.js

```diff
@@ -1,28 +1,36 @@
 module.exports = {
-  parser: 'babel-eslint',
-  extends: ['standard', 'standard-react', 'prettier', 'react-app'],
-  plugins: ['prettier', 'react'],
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'standard',
+    'plugin:prettier/recommended',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: ['react', 'prettier'],
   rules: {
-    camelcase: 'off',
+    'prettier/prettier': 'error',
     'react/prop-types': 'off',
-    'react/react-in-jsx-scope': 'off',
```

Comment:
Same goes for this rules - they are here for a reason 🤔

Reply:
Redundant